docs/guides/event_loop.md

```diff
@@ -2,14 +2,76 @@
 
 # An Introduction to the Event Loop in napari
 
+
+## tldr;
+
+It is not necessary to have a deep understanding of Qt or event loops to use
+napari.  napari attempts to use "sane defaults" for most scenarios. Here are the
+most important details:
+
+### In IPython or Jupyter Notebook
+
+napari will detect if you are running an an IPython or Jupyter shell, and will
+automatically use the [IPython GUI event
+loop](https://ipython.readthedocs.io/en/stable/config/eventloops.html#integrating-with-gui-event-loops).
+As of [version 0.4.7](https://github.com/napari/napari/releases/tag/v0.4.7) is
+no longer necessary to call `%gui qt` manually.  Just create a viewer:
+
+```python
+In [1]: import napari
+
+In [2]: viewer = napari.Viewer()  # viewer will show in a new window
+
+In [3]: ... # continue interactive usage
```

Comment:
I suggest we capitalize comments?

It kinda feels like a cleaner convention (that we do not enforce really... but we could :) )


so 

```python

# Continue interactive usage

vs

# continue interactive usage

```